HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -8,366 +10,382 @@ public enum Hug2Status
-        { "9079", new LocalAuthorityDetails("West Lothian Council", Hug2Status.NotTakingPart, "https://www.westlothian.gov.uk/") },
-        { "2845", new LocalAuthorityDetails("West Northamptonshire Council", Hug2Status.Pending, "https://www.westnorthants.gov.uk/") },
-        { "3125", new LocalAuthorityDetails("West Oxfordshire District Council", Hug2Status.Pending, "https://www.westoxon.gov.uk/") },
-        { "3545", new LocalAuthorityDetails("West Suffolk District Council", Hug2Status.Pending, "https://www.westsuffolk.gov.uk/") },
-        { "9020", new LocalAuthorityDetails("Comhairle nan Eilean Siar", Hug2Status.NotTakingPart, "https://www.cne-siar.gov.uk/") },
-        { "935", new LocalAuthorityDetails("Westmorland and Furness Council", Hug2Status.NotTakingPart, "https://www.westmorlandandfurness.gov.uk/") },
-        { "4250", new LocalAuthorityDetails("Wigan Metropolitan Borough Council", Hug2Status.NotTakingPart, "http://www.wigan.gov.uk") },
-        { "3940", new LocalAuthorityDetails("Wiltshire Council", Hug2Status.Pending, "https://www.wiltshire.gov.uk/") },
-        { "1765", new LocalAuthorityDetails("Winchester City Council", Hug2Status.Pending, "https://www.winchester.gov.uk/") },
-        { "355", new LocalAuthorityDetails("Royal Borough of Windsor and Maidenhead", Hug2Status.Pending, "http://www.rbwm.gov.uk/") },
-        { "4325", new LocalAuthorityDetails("Wirral Council", Hug2Status.Pending, "http://www.wirral.gov.uk/") },
-        { "3655", new LocalAuthorityDetails("Woking Borough Council", Hug2Status.Pending, "https://www.woking.gov.uk/") },
-        { "360", new LocalAuthorityDetails("Wokingham Borough Council", Hug2Status.Pending, "https://www.wokingham.gov.uk/") },
-        { "4635", new LocalAuthorityDetails("City of Wolverhampton Council", Hug2Status.Pending, "http://www.wolverhampton.gov.uk") },
-        { "1835", new LocalAuthorityDetails("Worcester City Council", Hug2Status.Pending, "http://www.worcester.gov.uk") },
-        { "3835", new LocalAuthorityDetails("Worthing Borough Council", Hug2Status.Pending, "https://www.adur-worthing.gov.uk/") },
-        { "6955", new LocalAuthorityDetails("Wrexham County Borough Council", Hug2Status.NotTakingPart, "http://www.wrexham.gov.uk/") },
-        { "1840", new LocalAuthorityDetails("Wychavon District Council", Hug2Status.Pending, "http://www.wychavon.gov.uk/") },
-        { "2370", new LocalAuthorityDetails("Wyre Council", Hug2Status.Pending, "http://www.wyre.gov.uk") },
-        { "1845", new LocalAuthorityDetails("Wyre Forest District Council", Hug2Status.Pending, "https://www.wyreforestdc.gov.uk/") },
-        { "2741", new LocalAuthorityDetails("City of York Council", Hug2Status.Pending, "https://www.york.gov.uk/") },
+        { "9052", new LocalAuthorityDetails("Aberdeenshire Council", Hug2Status.NotTakingPart, "https://www.aberdeenshire.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
```

Comment:
Do we expect to see examples with more exotic sets of options? It feels like it might be slightly more natural to store the `IncomeThreshold` here and map to the options when required.

This would give us a bit more freedom to handle different thresholds, e.g. if we need to display different help text on the page for the different thresholds, or handle cases which aren't just a simple set of options. You can toggle behaviour based on `localAuthority.IncomeThreshold` rather than having to introspect on the available options.

Also happy for you to tackle that problem if and when it arises 🙂

Reply:
For now we've been assured that it is always a single threshold with above/below options. The actual rules have turned out to be so complicated that we aren't going to try to put them fully on the page anyway, so we probably won't need to change wording after all.
So I'll leave this as is for now, we can always change it if we need to.